esmvaltool/diag_scripts/impact/quick_insights.py

```diff
@@ -0,0 +1,210 @@
+
+        xrds = xr.open_dataset(input_file)
+        xrda = xrds[short_name]
+
+        # Make sure datasets can be combined
+        make_standard_calendar(xrda)
+        redundant_dims = np.setdiff1d(xrda.coords, xrda.dims)
+        xrda = xrda.drop(redundant_dims)
+
+        data_arrays.append(xrda)
+        identifiers.append(alias)
+        ancestors.append(input_file)
+
+    # Combine along a new dimension
+    data_array = xr.concat(data_arrays, dim='dataset')
+    data_array['dataset'] = identifiers
+
+    return data_array, ancestors
+
+
+def area_weighted_mean(data_array: 'xr.DataArray') -> 'xr.DataArray':
+    """Calculate area mean weighted by the latitude."""
+    weights_lat = np.cos(np.radians(data_array.lat))
+    means = data_array.weighted(weights_lat).mean(dim=['lat', 'lon', 'time'])
+
+    return means
+
+
+def calculate_bias(model_data: 'xr.DataArray',
+                   obs_data: 'xr.DataArray') -> 'xr.DataArray':
+    """Calculate area weighted RMSD with respect to (mean of) observations."""
+    if len(obs_data['dataset']) > 1:
+        obs_data = obs_data.mean(dim='dataset')
+    else:
+        obs_data = obs_data.squeeze()
+
+    diff = model_data - obs_data
+    bias = area_weighted_mean(diff**2)**0.5
+
+    bias.attrs = model_data.attrs
+    return bias
+
+
+def plot_scatter(tidy_df, ancestors, cfg):
+    """Plot bias on one axis and change on the other."""
+    grid = sns.relplot(
+        data=tidy_df,
+        x="bias",
+        y="change",
+        hue="dataset",
+        col="variable",
+        facet_kws=dict(sharex=False, sharey=False),
+        kind='scatter',
+    )
+
+    filename = get_plot_filename('bias_vs_change', cfg)
+    grid.fig.savefig(filename, bbox_inches='tight')
+
+    caption = "Bias and change for each variable"
+    log_provenance(filename, ancestors, caption, cfg)
+
+
+def plot_table(tidy_df, ancestors, cfg):
+    """Render pandas table as a matplotlib figure."""
+    fig, axes = plt.subplots()
+    pd.plotting.table(axes, tidy_df.reset_index().round(2))
+    axes.set_axis_off()
+
+    filename = get_plot_filename('table', cfg)
+    fig.savefig(filename, bbox_inches='tight')
+
+    caption = "Bias and change for each variable"
+    log_provenance(filename, ancestors, caption, cfg)
+
+
+def plot_htmltable(tidy_df, ancestors, cfg):
+    """Render pandas table as html output.
+
+    # https://pandas.pydata.org/pandas-docs/stable/user_guide/style.html
+    """
+    styled_table = (tidy_df.unstack('variable').style.background_gradient(
+        cmap='RdYlGn', low=.2, high=1, axis=0).format("{:.2e}",
+                                                      na_rep="-").render())
+    filename = get_diagnostic_filename('bias_vs_change', cfg, extension='html')
+    with open(filename, 'w') as htmloutput:
+        htmloutput.write(styled_table)
+
+    caption = "Bias and change for each variable"
+    log_provenance(filename, ancestors, caption, cfg)
+
+
+def main(cfg):
```

Comment:
This needs a docstring.

```suggestion
def main(cfg):
```